modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ContainerTag.java

```diff
@@ -56,7 +56,7 @@ protected String processCssClasses(Set<String> cssClasses) {
 		else {
 			cssClasses.add("container-fluid");
 
-			if (Validator.isNotNull(_size)) {
+			if (Validator.isNotNull(_size) && (_size != "false")) {
```

Comment:
I think this check should actually be done in ContainerFluidTag.

Reply:
I had to use `Validator.isNotNull(_size) && !(_size == "false")` here. I ran into trouble